infra/main.bicep

```diff
@@ -82,7 +82,7 @@ param azureOpenAIModelVersion string = '0613'
   'openai_function'
   'langchain'
 ])
-param orchestrationStrategy string = 'langchain'
+param orchestrationStrategy string = 'openai_function'
```

Comment:
How come?

Reply:
The documentation states that we should be using `openai_function` if using 613 (which is the default) - so just aligning that.